js/views/userCard.js

```diff
@@ -0,0 +1,213 @@
+import $ from 'jquery';
+import _ from 'underscore';
+import BaseVw from './baseVw';
+import loadTemplate from '../utils/loadTemplate';
+import app from '../app';
+import { followedByYou, followUnfollow } from '../utils/follow';
+import Profile from '../models/profile/Profile';
+import UserCard from '../models/UserCard';
+import { launchModeratorDetailsModal } from '../utils/modalManager';
+import { openSimpleMessage } from './modals/SimpleMessage';
+
+export default class extends BaseVw {
+  constructor(options = {}) {
+    super(options);
+    this.options = options;
+
+    if (this.model instanceof Profile) {
+      this.guid = this.model.id;
+      this.fetched = true;
+    } else if (this.model) {
+      this.guid = this.model.get('guid');
+      this.fetched = false;
+    } else {
+      this.model = new UserCard({ guid: options.guid });
+      this.guid = options.guid;
+      this.fetched = false;
+    }
+
+    if (!this.guid) {
+      if (this.model) {
+        throw new Error('The guid must be provided in the model.');
+      } else {
+        throw new Error('The guid must be provided in the options.');
+      }
+    }
+
+    this.ownGuid = this.guid === app.profile.id;
+    this.followedByYou = followedByYou(this.guid);
+
+    this.loading = !this.fetched;
+    this.settings = app.settings.clone();
+
+    this.listenTo(this.settings, 'sync', () => {
+      app.settings.set(this.settings.toJSON());
+      this.$modBtn.toggleClass('active', this.ownMod);
```

Comment:
For toggling of the active class, I would listen to `change:storeModerators` on app.settings. That way if the change was initiated outside of here (e.g. the store tab in the settings modal), the button will reflect the correct state.